authapi/authapi.go

```diff
@@ -19,19 +19,9 @@ func NewAuthApi(api duoapi.DuoApi) *AuthApi {
 	return &AuthApi{api}
 }
 
-// API calls will return a StatResult object.  On success, Stat is 'OK'.
-// On error, Stat is 'FAIL', and Code, Message, and Message_Detail
-// contain error information.
-type StatResult struct {
```

Comment:
Removing this exported field will technically break backwards compatibility. I think it's OK to introduce a struct with the same name in the duoapi module and use it everywhere, but this struct should be left here (and maybe a comment describing what's happening.)
`/* Leaving for backwards compatibility. The struct in use has been moved to duoapi, to be shared between the auth and admin modules.*/
type StatResult struct {Stat           string
	Code           *int32
	Message        *string
	Message_Detail *string
}`

Reply:
Done!